### PR TITLE
Publications added for EMSE submission

### PR DESCRIPTION
**Pull request by SundermannC**

### CODE REVIEW

File: literature.bib [hunk 1]
```diff
@@ -107,6 +107,37 @@ @mastersthesis{WitschelProject21
 	year = 2021
 }
 
+@article{FHH:JEA21,
+	author = {Fichte, Johannes K. and Hecher, Markus and Hamiti, Florim},
+	title = {{The Model Counting Competition 2020}},
+	year = 2021,
+	issue_date = {December 2021},
```
Comment (pmbittner) [inline literature.bib:114]: I am not sure bit I believe we usually omit this entry as it is redundant.

File: literature.bib [hunk 2]
```diff
@@ -107,6 +107,37 @@ @mastersthesis{WitschelProject21
...
+	url = {https://doi.org/10.1145/3459080},
+	doi = {10.1145/3459080},
+	journal = JEA,
+	month = OCT,
+	articleno = {13},
+	numpages = {26}
+}
+
+
+
```
Comment (pmbittner) [inline literature.bib:128]: much whitespace here

File: literature.bib [hunk 3]
```diff
@@ -107,6 +107,37 @@ @mastersthesis{WitschelProject21
...
+% issue and pages still missing, maybe because it is open access
+@article{NST+:SoSyM21,
+	author = {Nieke, Michael and Sampaio, Gabriela and Th{\"u}m, Thomas and Seidl, Christoph and Teixeira, Leopoldo and Schaefer, Ina},
+	title = {{Guiding the Evolution of Product-Line Configurations}},
+	journal = SoSyM,
+	pages = {1--23},
+	doi = {https://link.springer.com/article/10.1007/s10270-021-00906-w},
```
Comment (pmbittner) [inline literature.bib:135]: This is not the DOI but an URL.

File: literature.bib [hunk 4]
```diff
@@ -490,6 +521,15 @@ @inproceedings{SL:VaMoS21
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2020 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@techreport{OGB+:TR20,
+  title={Scalable Uniform Sampling for Real-World Software Product Lines},
+  author={Oh, Jeho and Gazzillo, Paul and Batory, Don and Heule, Marijn and Myers, Margaret},
+  year= 2020,
+  institution = {The University of Texas at Austin, Department of Computer Science},
+  number = {TR-20-01}
+}
```
Comment (pmbittner) [inline literature.bib:530]: inconsistent indentation and style

File: literature.bib [hunk 5]
```diff
@@ -490,6 +521,15 @@ @inproceedings{SL:VaMoS21
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2020 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@techreport{OGB+:TR20,
+  title={Scalable Uniform Sampling for Real-World Software Product Lines},
```
Comment (pmbittner) [inline literature.bib:525]: {{}}

File: literature.bib [hunk 6]
```diff
@@ -879,6 +919,17 @@ @inproceedings{FHS:CP20
...
+
+@inproceedings{DPV:AAAI20,
+  title={{ADDMC: Weighted Model Counting with Algebraic Decision Diagrams}},
+  author={Dudek, Jeffrey and Phan, Vu and Vardi, Moshe},
+  booktitle=AAAI,
+  volume={34},
+  number={02},
+  pages={1468--1476},
+  year={2020}
+}
```
Comment (pmbittner) [inline literature.bib:931]: inconsistent style

File: literature.bib [hunk 7]
```diff
@@ -879,6 +919,17 @@ @inproceedings{FHS:CP20
 	year = 2020
 }
 
+
+@inproceedings{DPV:AAAI20,
+  title={{ADDMC: Weighted Model Counting with Algebraic Decision Diagrams}},
+  author={Dudek, Jeffrey and Phan, Vu and Vardi, Moshe},
+  booktitle=AAAI,
+  volume={34},
+  number={02},
```
Comment (pmbittner) [inline literature.bib:928]: remove preceding 0?

File: literature.bib [hunk 8]
```diff
@@ -1111,6 +1162,16 @@ @article{LML:SCP19
...
 
+@inproceedings{FHZ:CP19,
+	author={Fichte, Johannes K. and Hecher, Markus and Zisser, Markus},
+	title= {{An Improved GPU-Based SAT Model Counter}},
+	booktitle=CP,
+	year= 2019,
+	publisher=Springer,
+	address=Cham,
+	pages={491--509}
+}
```
Comment (pmbittner) [inline literature.bib:1173]: s.a.

File: literature.bib [hunk 9]
```diff
@@ -5234,6 +5295,16 @@ @inproceedings{SSA:ICSE15
...
 }
 
+@inproceedings{BAM:ICSR15,
+	title= {{Measures for Quality Evaluation of Feature Models}},
+	author= {Bezerra, Carla I. M. and Andrade, Rossana M. C. and Monteiro, Jos{\'e} Maria S.},
+	booktitle = ICSR,
+	publisher = Springer,
+	address = BerlinHeidelberg,
+	year= 2014,
+	pages= {282--297}
```
Comment (pmbittner) [inline literature.bib:5305]: s.a.

File: literature.bib (hunk 6 above)
Comment (pmbittner) [reply literature.bib:931]: publisher and address missing

File: literature.bib [hunk 10]
```diff
@@ -13013,6 +13084,15 @@ @article{ALMK:SCP10
 	year = 2010
 }
 
+@article{MN:CEP2010,
+  title={{Mann-Whitney U Test}},
+  author={McKnight, Patrick E and Najab, Julius},
+  journal={The Corsini Encyclopedia of Psychology},
+  pages={1--1},
+  year=2010,
+  publisher={Wiley Online Library}
```
Comment (pmbittner) [inline literature.bib:13093]: I believe we have a constant for this publisher. Also inconsistent style.

File: literature.bib [hunk 11]
```diff
@@ -13479,6 +13559,16 @@ @article{HJS:JSAT09
...
 }
 
+@inproceedings{ABL:CP09,
+	author={Ans{\'o}tegui, Carlos and Bonet, Mar{\'i}a Luisa and Levy, Jordi},
+	title={{On the Structure of Industrial SAT Instances}},
+	booktitle=CP,
+	year= 2009,
+	publisher=Springer,
+	address=BerlinHeidelberg,
+	pages= {127--141},
```
Comment (pmbittner) [inline literature.bib:13569]: s.a.

File: literature.bib [hunk 12]
```diff
@@ -18086,6 +18176,18 @@ @incollection{R03
...
+@article{AVM:SAGE02,
+  title={{Bravais-Pearson and Spearman Correlation Coefficients: Meaning, Test of Hypothesis and Confidence Interval}},
+  author={Artusi, R and Verderio, P and Marubini, EJTIjobm},
+  journal={The International journal of biological markers},
+  volume={17},
+  number={2},
+  pages={148--151},
+  year=2002,
+  publisher={SAGE Publications Sage UK: London, England}
+}
```
Comment (pmbittner) [inline literature.bib:18188]: Githubs syntax highlighting stopped working here. Might there be a syntax somewhere above this entry? Also inconsistent style

File: literature.bib [hunk 13]
```diff
@@ -18086,6 +18176,18 @@ @incollection{R03
...
 
+@article{AVM:SAGE02,
+  title={{Bravais-Pearson and Spearman Correlation Coefficients: Meaning, Test of Hypothesis and Confidence Interval}},
+  author={Artusi, R and Verderio, P and Marubini, EJTIjobm},
+  journal={The International journal of biological markers},
+  volume={17},
+  number={2},
+  pages={148--151},
+  year=2002,
+  publisher={SAGE Publications Sage UK: London, England}
```
Comment (pmbittner) [inline literature.bib:18187]: is `London, England` part of the publisher's name? If not put in address?

File: literature.bib [hunk 14]
```diff
@@ -18086,6 +18176,18 @@ @incollection{R03
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2002 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@article{AVM:SAGE02,
+  title={{Bravais-Pearson and Spearman Correlation Coefficients: Meaning, Test of Hypothesis and Confidence Interval}},
+  author={Artusi, R and Verderio, P and Marubini, EJTIjobm},
+  journal={The International journal of biological markers},
```
Comment (pmbittner) [inline literature.bib:18182]: inconsistent capitalization

File: literature.bib [hunk 15]
```diff
@@ -21129,6 +21231,18 @@ @article{MF:TSE81
...
+@article{CI:AS81,
+  title= {{Rank Transformations as a Bridge Between Parametric and Nonparametric Statistics}},
+  author={Conover, William J and Iman, Ronald L},
+  journal={The American Statistician},
+  volume={35},
+  number={3},
+  pages={124--129},
+  year=1981,
+  publisher=TaylorFrancis
+}
```
Comment (pmbittner) [inline literature.bib:21243]: s.a.

File: literature.bib [hunk 16]
```diff
@@ -21716,6 +21830,18 @@ @ARTICLE{KMSL:CDT83
...
+@article{Z:JASA72,
+  title={{Significance Testing of the Spearman Rank Correlation Coefficient}},
+  author={Zar, Jerrold H},
+  journal={Journal of the American Statistical Association},
+  volume={67},
+  number={339},
+  pages={578--580},
+  year=1972,
+  publisher={Taylor \& Francis}
+}
```
Comment (pmbittner) [inline literature.bib:21842]: s.a.

**Review by pmbittner — CHANGES_REQUESTED** [review]

**Comment by pmbittner** [comment]
The month is also missing for many entries but I guess because these are not our own papers, that should be ok.

File: literature-cleaned.bib [hunk 17]
```diff
@@ -8448,6 +8516,18 @@ @inproceedings{ACG+:VaMoS12
 	year = 2012
 } 
 
+
+@article{SF:JGME12,
+	title = {{Using Effect Size—or Why the P Value is Not Enough}},
+	author = {Sullivan, Gail M and Feinn, Richard},
+	journal = {Journal of Graduate Medical Education},
```
Comment (pmbittner) [inline literature-cleaned.bib:8523]: add constant? Is JGME the official abbreviation?

File: literature-cleaned.bib [hunk 18]
```diff
@@ -8952,6 +9032,16 @@ @inproceedings{STKS:VaMoS12
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2011 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@inproceedings{D:IJCAI11,
+	author = {Darwiche, Adnan},
+	title = {SDD: A New Canonical Representation of Propositional Knowledge Bases},
```
Comment (pmbittner) [inline literature-cleaned.bib:9037]: double bracket

File: literature-cleaned.bib [hunk 19]
```diff
@@ -8952,6 +9032,16 @@ @inproceedings{STKS:VaMoS12
...
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2011 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@inproceedings{D:IJCAI11,
+	author = {Darwiche, Adnan},
+	title = {SDD: A New Canonical Representation of Propositional Knowledge Bases},
+	year = 2011,
+	publisher = {AAAI Press},
+	pages = {819--826},
+	numpages = {8},
+	series = {IJCAI'11}
```
Comment (pmbittner) [inline literature-cleaned.bib:9042]: no series

File: literature-cleaned.bib [hunk 20]
```diff
@@ -17243,6 +17363,18 @@ @article{MF:TSE81
 	number={5},
 }
 
+@article{CI:AS81,
+	title= {{Rank Transformations as a Bridge Between Parametric and Nonparametric Statistics}},
+	author = {Conover, William J and Iman, Ronald L},
+	journal = {The American Statistician},
```
Comment (pmbittner) [inline literature-cleaned.bib:17369]: s.a.

File: literature-cleaned.bib [hunk 21]
```diff
@@ -10411,6 +10501,15 @@ @article{ALMK:SCP10
 	year = 2010
 }
 
+@article{MN:CEP2010,
+	title = {{Mann-Whitney U Test}},
+	author = {McKnight, Patrick E and Najab, Julius},
+	journal = {The Corsini Encyclopedia of Psychology},
```
Comment (pmbittner) [inline literature-cleaned.bib:10507]: s.a.

File: literature-cleaned.bib [hunk 22]
```diff
@@ -17742,6 +17874,17 @@ @ARTICLE{KMSL:CDT83
 	year = 1983
 }
 
+@article{Z:JASA72,
+  title = {{Significance Testing of the Spearman Rank Correlation Coefficient}},
+  author = {Zar, Jerrold H},
+  journal = {Journal of the American Statistical Association},
```
Comment (pmbittner) [inline literature-cleaned.bib:17880]: s.a.

File: literature-cleaned.bib [hunk 23]
```diff
@@ -17742,6 +17874,17 @@ @ARTICLE{KMSL:CDT83
...
 
+@article{Z:JASA72,
+  title = {{Significance Testing of the Spearman Rank Correlation Coefficient}},
+  author = {Zar, Jerrold H},
+  journal = {Journal of the American Statistical Association},
+  volume = {67},
+  number = {339},
+  pages = {578--580},
+  year = 1972,
+  publisher = {Taylor \& Francis}
```
Comment (pmbittner) [inline literature-cleaned.bib:17885]: use constant

File: literature-cleaned.bib (hunk 21 above)
Comment (SundermannC) [reply literature-cleaned.bib:10507]: Could not find an official abbreviation.

File: literature-cleaned.bib (hunk 20 above)
Comment (SundermannC) [reply literature-cleaned.bib:17369]: Could not find an official abbreviation

Comment (pmbittner) [reply literature-cleaned.bib:17357]: Then leave as is and probably add a comment that you could not find it, so nobody else starts searching for it again.

**Review by pmbittner — APPROVED** [review]

**Pull request merged**